docs/develop/create-addin-commands.md

```diff
@@ -290,17 +192,41 @@ A button performs a single action when the user selects it. It can either execut
-|**\<Supertip\>** <br/> | Required. The supertip for this button, which is defined by the following: <br/> **Title** <br/>  Required. The text for the supertip. The **resid** attribute must be set to the value of the **id** attribute of a **\<String\>** element. The **\<String\>** element is a child element of the **\<ShortStrings\>** element, which is a child element of the **\<Resources\>** element. <br/> **\<Description\>** <br/>  Required. The description for the supertip. The **resid** attribute must be set to the value of the **id** attribute of a **\<String\>** element. The **\<String\>** element is a child element of the **\<LongStrings\>** element, which is a child element of the **\<Resources\>** element. <br/> |
-|**\<Icon\>** <br/> | Required. Contains the **\<Image\>** elements for the button. Image files must be .png format. <br/> **\<Image\>** <br/>  Defines an image to display on the button. The **resid** attribute must be set to the value of the **id** attribute of an **\<Image\>** element. The **\<Image\>** element is a child element of the **\<Images\>** element, which is a child element of the **\<Resources\>** element. The **size** attribute indicates the size, in pixels, of the image. Three image sizes are required: 16, 32, and 80. Five optional sizes are also supported: 20, 24, 40, 48, and 64. <br/> |
-|**\<Action\>** <br/> | Required. Specifies the action to perform when the user selects the button. You can specify one of the following values for the **xsi:type** attribute: <br/> **ExecuteFunction**, which runs a JavaScript function located in the file referenced by **\<FunctionFile\>**. The **\<FunctionName\>** child element specifies the name of the function to execute. <br/> **ShowTaskPane**, which shows the add-in's task pane. The **\<SourceLocation\>** child element specifies the source file location of the page to display. The **resid** attribute must be set to the value of the **id** attribute of a **\<Url\>** element in the **\<Urls\>** element in the **\<Resources\>** element. <br/> |
+The following code shows an example function used by **\<FunctionName\>**. Note the call to [`event.completed`](/javascript/api/office/office.addincommands.event#office-office-addincommands-event-completed-member(1)), This signals that you have successfully handled the event. When a function is called multiple times, such as multiple clicks on the same add-in command, all events are automatically queued. The first event runs automatically, while the other events remain on the queue. When your function calls `event.completed`, the next queued call to that function runs. You must implement `event.completed`, otherwise your function won't run.
```

Comment:
```suggestion
The following code shows an example function used by **\<FunctionName\>**. Note the call to [`event.completed`](/javascript/api/office/office.addincommands.event#office-office-addincommands-event-completed-member(1)). This signals that you've successfully handled the event. When a function is called multiple times, such as multiple clicks on the same add-in command, all events are automatically queued. The first event runs automatically, while the other events remain on the queue. When your function calls `event.completed`, the next queued call to that function runs. You must implement `event.completed`, otherwise your function won't run.
```